contracts/liquidity_hub/pool-manager/src/manager/commands.rs

```diff
@@ -108,8 +107,8 @@ pub fn create_pool(
     // send pool creation fee to whale lair
```

Comment:
still referencing whale lair here

Reply:
yeah i saw that today, once this PR is merged we will go through the contracts and cleanup some names in the other v2 contracts.